app/models/hud_reports/report_client_base.rb

```diff
@@ -55,8 +55,10 @@ def display_value(col, pii_policy:, include_content_tag: true)
         HudUtility2024.relationship_to_hoh(cell)
       when /.*disabling_condition$/
         HudUtility2024.disability_response(cell)
-      when 'first_name', 'last_name', 'middle_name', 'full_name', 'brief_name'
+      when /.*first_name/, /.*last_name/, /.*middle_name/, /.*full_name/, /.*brief_name/
```

Comment:
Should these end in `$`?
```suggestion
      when /.*first_name$/, /.*last_name$/, /.*middle_name$/, /.*full_name$/, /.*brief_name$/
```

Reply:
I don't think it would hurt. I'll do a quick test to be sure and can push it up.